### ISSUE TITLE
Breaking changes in orleans 3.1

### ISSUE THREAD

**Issue by Ilchert** [OrleansContrib/OrleansTestKit#86]
With a new version of Orleans (3.1.1), I got an error 
```
'Unable to load one or more of the requested types.
Method 'GetStorage' on type 'Orleans.TestKit.TestGrainRuntime' from assembly 'OrleansTestKit, Version=3.0.1.0, Culture=neutral, PublicKeyToken=null' tried to implicitly implement an interface method with weaker type parameter constraints.'
```
It looks like [this ](https://github.com/dotnet/orleans/commit/4531cd4438a7008617ec7aede78b27c5807a3151#diff-e3b54154434be59cc51ad629b908909e) change did breaking change in `IOrleansRuntime`.

**Comment by Hechamon** [comment]
Since we just ran into the same problem I created a PR to up the Orleans version.
[https://github.com/OrleansContrib/OrleansTestKit/pull/87](https://github.com/OrleansContrib/OrleansTestKit/pull/87)

**Comment by seniorquico** [comment]
Fixed in #87. An updated 3.1.0 package has been published to NuGet.
